hubblestack/audit/command_line_parser.py

```diff
@@ -457,3 +457,22 @@ def get_filtered_params_to_log(block_id, block_dict, extra_args=None):
         'key_aliases': key_aliases,
         'delimiter': delimiter
     }
+
+
+def get_failure_reason(block_id, block_dict, extra_args=None):
+    """
+
+    :param block_id:
+        id of the block
+    :param block_dict:
+        parameter for this module
+    :param extra_args:
+        Extra argument dictionary, (If any)
+        Example: {'chaining_args': {'result': {'cmdline': 'app --config-file=test test'}, 'status': True},
+                  'caller': 'Audit'}
+    :return:
+    """
+    key_aliases = runner_utils.get_param_for_module(block_id, block_dict, 'key_aliases')
+    command_line = runner_utils.get_param_for_module(block_id, block_dict, 'cmdline')
```

Comment:
We are not considering chaining params. As we did in execute(). 
Please check in all modules.

Reply:
As discussed, we will do this in future